roxctl/common/report/report.go

```diff
@@ -70,6 +70,24 @@ func JSON(output io.Writer, alerts []*storage.Alert) error {
 	return nil
 }
 
+// JSONRemarks pipes out the violated alerts and remarks as JSON.
+func JSONRemarks(output io.Writer, alerts []*storage.Alert, remarks []*v1.DeployDetectionRemark) error {
+	// This object is really just a filler because it's a wrapper around the alerts
+	// this is required because jsonpb can only marshal proto.Message
+	bdr := &v1.ResultAggregation{
+		Alerts:  alerts,
+		Remarks: remarks,
+	}
+	marshaler := jsonpb.Marshaler{Indent: "  "}
+	if err := marshaler.Marshal(output, bdr); err != nil {
+		return errors.Wrap(err, "could not marshal alerts")
+	}
+	if _, err := output.Write([]byte{'\n'}); err != nil {
+		return errors.Wrap(err, "could not write alerts")
+	}
```

Comment:
The error reads a bit strange - you try only to write a newline and error about inability to write alerts?

Reply:
The functionality is the same as it is for `JSON` (see line 55,ff.). I was thinking to refactor both of them, but decided to leave the duplication in as they are very small.
I've updated the error message to something more readable.